### PR TITLE
additive product proof fixed

### PR DESCRIPTION
**Pull request by 0x6980**
I implemented the code of this part and this pull request is correction of the algorithm

### CODE REVIEW

File: content/range-proof/en/range-proof.md [hunk 1]
```diff
@@ -186,7 +186,7 @@ Note that $\mathbf{k}$ is not Hadamard multiplied with $\mathbf{y}^n$, but the l
 
 For now, we compute $t(x)$ as
 
-$$t(x)=v+t_1x+t_2x$$
+$$t(x)=vz+t_1x+t_2x$$
```
Comment (tanim0la) [inline content/range-proof/en/range-proof.md:190]: it is stated below that the constant term in t(x) is `v` and not `vz`

File: content/range-proof/en/range-proof.md [hunk 2]
```diff
@@ -186,7 +186,7 @@ Note that $\mathbf{k}$ is not Hadamard multiplied with $\mathbf{y}^n$, but the l
...
```
Comment (tanim0la) [inline content/range-proof/en/range-proof.md]: Also, `t_2x` should be squared (`t_2x^2`)

Comment (0x6980) [reply content/range-proof/en/range-proof.md]: yeah, that is true. I will fix it

**Review by tanim0la — APPROVED** [review]

**Review by RareSkills — APPROVED** [review]

**Comment by RareSkills** [comment]
Good stuff, appreciate it!

**Pull request merged**